pdf/nodejs/acroforms/set/_index.md

```diff
@@ -0,0 +1,193 @@
+
+{{% blocks/products/pf/agp/content h2="Set an AcroForms from PDF document via Cloud Node.js SDK" %}}
+
+In order to set an AcroForms from PDF, we'll use
+[Aspose.PDF Cloud Node.js SDK](https://products.aspose.cloud/pdf/nodejs/). This Cloud SDK assists Node.js programmers in developing cloud-based PDF creator, annotator, editor, and converter apps using Node.js programming language via Aspose.PDF REST API. Simply create an account at [Aspose for Cloud](https://dashboard.aspose.cloud/#/apps) and get your application information. Once you have the App SID & key, you are ready to give the Aspose.PDF Cloud Node.js SDK.
+
+{{% blocks/products/pf/agp/code-block title="Package Manager Console Command" offSpacer="true" %}}
+
+```bash
+
+     
+    npm install asposepdfcloud --save
+     
+     
+
+```
+
+{{% /blocks/products/pf/agp/code-block %}}
+
+{{% /blocks/products/pf/agp/content %}}
+
+{{< blocks/products/pf/agp/feature-section isGrey="true" >}}
+
+{{% blocks/products/pf/agp/feature-section-col title="Steps to set an AcroForms from PDF via Cloud Node.js" %}}
+
+{{% blocks/products/pf/agp/text %}}
+
+Aspose.PDF Cloud developers can easily load & set an acroforms from PDF in just a few lines of code.
+
+{{% /blocks/products/pf/agp/text %}}
+
+1. Read the local PDF file
+1. Create a field with new decorations
+1. Update the form field in the uploaded PDF
+1. Download the result if needed it
+
+{{% /blocks/products/pf/agp/feature-section-col %}}
+
+{{% blocks/products/pf/agp/feature-section-col title="System Requirements" %}}
+
+{{% blocks/products/pf/agp/text %}}
+
+It is easy to get started with Aspose.PDF Cloud Node.js SDK and there is nothing to install. Simply create an account at Aspose for Cloud and get your application information. Once you have the App SID & key, you are ready to give the Aspose.PDF Cloud Node.js SDK.
+
+* CPU: 1GHz
+* RAM: 512Mb
+* Free space on disk: 20Mb
+* Any OS Microsoft Windows, Mac OS, Linux x32/x64
+* node.js v4.8 or higher
+
+{{% /blocks/products/pf/agp/text %}}
+
+{{% /blocks/products/pf/agp/feature-section-col %}}
+
+{{% blocks/products/pf/agp/code-block title="This sample code shows setting an AcroForms from PDF documents - Node.js" offSpacer="" %}}
+
+```js
+
+    import credentials from "./credentials.json" with { type: "json" };
+    import fs from 'node:fs/promises';
+    import { PdfApi } from "asposepdfcloud";
+    import { Border } from "asposepdfcloud/src/models/border";
+    import { TextBoxField } from "asposepdfcloud/src/models/textBoxField";
+    import { Dash } from "asposepdfcloud/src/models/dash";
+    import { Rectangle } from "asposepdfcloud/src/models/rectangle";
+    import { Field } from "asposepdfcloud/src/models/field";
+    import { FieldType } from "asposepdfcloud/src/models/fieldType";
+    import { Fields } from "asposepdfcloud/src/models/fields";
```

Comment:
Please create a task for yourself and check this example later with version 25.2/25.3. It seems they have a breaking changes